### PR TITLE
Convert toggles into docsify tabs

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Convert toggles into docsify tabs.
- Use lowercase for 'custom resource'.

**Related issue(s)**
See also [#18379](https://github.com/kyma-project/kyma/issues/18379)

### CODE REVIEW

File: docs/contributor/02-10-operations.md [hunk 1]
```diff
@@ -133,5 +133,5 @@ Only one Condition of type `Ready` is used.
 
 ## Updating
 
-The update process is almost the same as the provisioning process. The only difference is BtpOperator CR existence in the cluster. 
-For the update process, the custom resource should be present in the cluster with the `Ready` state.  
+The update process is almost the same as the provisioning process. The only difference is BtpOperator custom resource (CR) existence in the cluster.
```
Comment (nataliasitko) [inline docs/contributor/02-10-operations.md:136]: Wouldn't it be better to explain the abbreviation the first time it is mentioned in the doc? https://github.com/kyma-project/btp-manager/pull/504/files#diff-00af6b14c6b2db6dc0c3738b429e323577e59f59eaabb0cc2c32bc11055a84b8L30

Comment (IwonaLanger) [reply docs/contributor/02-10-operations.md:136]: > Wouldn't it be better to explain the abbreviation the first time it is mentioned in the doc? https://github.com/kyma-project/btp-manager/pull/504/files#diff-00af6b14c6b2db6dc0c3738b429e323577e59f59eaabb0cc2c32bc11055a84b8L30

Of course. I missed this one. Thank you for spotting it 😄

**Review by MarekMichali — DISMISSED** [review]

**Review by MarekMichali — APPROVED** [review]

**Pull request merged**